config/default.yml

```diff
@@ -36,7 +36,7 @@ AllCops:
   # -S/--display-style-guide option.
   DisplayStyleGuide: false
   # Extra details are not displayed in offense messages by default. Change
-  # behaviour by overriding ExtraDetails, or by giving the
```

Comment:
This was intentional. :-) I don't like American English, but I respect its dominion over codebases.